cell.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CELL_H
+#define CELL_H
+#include <iostream>
+#include <map>
+#include <string>
+#include "direction.h"
+
+class Cell{
+public:
+    Cell(){resetCell();}
+    Cell(int v);
+    bool cellIsMovable(); //Check if the cell is movable
+    bool haveWinValue(){return value >= 2048;} //Check if the cell contains win value (2048)
+    int getCellValue(){return value;}
```

Comment:
No need of Cell here since it is a class, getValue or just value()

Reply:
just a value() is better

Reply:
Also getter functions must be const

Reply:
done